packages/transaction-controller/src/utils.ts

```diff
@@ -60,38 +62,65 @@ export function validateTxParams(
   txParams: TransactionParams,
   isEIP1559Compatible = true,
 ) {
-  if (
-    !txParams.from ||
-    typeof txParams.from !== 'string' ||
-    !isValidHexAddress(txParams.from)
-  ) {
-    throw rpcErrors.invalidParams(
-      `Invalid "from" address: ${txParams.from} must be a valid string.`,
-    );
-  }
+  validateEIP1559Compatibility(txParams, isEIP1559Compatible);
+  validateFrom(txParams);
```

Comment:
As we don't need any other properties from the params, should we just pass the `from` property directly? Same for the recipient below also.

Reply:
Sure for simplicity now it passes only the `from` property.